### PR TITLE
Add range support for `S01E01-02`

### PR DESCRIPTION
**Pull request by Jephuff**
This might be more of a conversation started, I needed support for multiple episodes.

this adds that in the form of a `range` so you get the first episode and the number of extra episodes
`S01E01-02` => `{episode: 1, range: 1}`
`S01E01-03` => `{episode: 1, range: 2}`

### CODE REVIEW

**Comment by tregusti** [comment]
I like the concept, but not the way the data is available to the consumer. `range` is not enough as a descriptive name, since it could mean the year range, or season range. It's not clear that is related to episode numbering, and it's not future proof, since adding another data set range later is then not so easy.

Also, `range` is not very clear. When the file covers 3 episodes, range is 2. I would expect there to be a property with a value of 3. The question is what the name of it should be. `episodeCount`?

**Comment by Jephuff** [comment]
yeah very good points. I like episodeCount, so it could be be 1 normally?
```
S01E01 => {episode: 1, episodeCount: 1}
S01E01-02 => {episode: 1, episodeCount: 2}
S01E01-03 => {episode: 1, episodeCount: 3}
```

**Comment by tregusti** [comment]
I like it.

But could you please add it to the other parsers for completeness as well?

File: lib/parser.js [hunk 1]
```diff
@@ -29,24 +29,26 @@ function parseEnglishLike(filename) {
...
 
     return result
   }
 }
 
 function parseSceneLikeWithX(filename) {
-  var m, result = {}
+  var m, result = { episodeCount: 1 }
 
   if (m = filename.match(/^(.+?)([\. ])(?:((?:20|19)\d\d)\2)?(\d+)x(\d+)\2/i)) {
```
Comment (tregusti) [inline lib/parser.js]: Here we can add episode count parsing as well. I assume with `-dd` format.

File: lib/parser.js [hunk 2]
```diff
@@ -62,7 +64,7 @@ function parseSceneLikeWithX(filename) {
   }
 }
 function parseSceneLike000(filename) {
-  var m, result = {}
+  var m, result = { episodeCount: 1 }
 
   if (m = filename.match(/^(.+?)([\. ])(?:((?:20|19)\d\d)\2)?(\d\d\d+)\2/i)) {
```
Comment (tregusti) [inline lib/parser.js]: Here I'm more uncertain if the common format would be `-dd` as well. Strange format with `xxx…` anyway. =)

**Review by tregusti — CHANGES_REQUESTED** [review]

File: lib/parser.js (hunk 2 above)
Comment (Jephuff) [reply lib/parser.js]: yeah definitely not sure about this one haha
-dd seems safe though?

**Comment by tregusti** [comment]
Thank you!

**Pull request merged**